src/dawn/Compiler/Options.inc

```diff
@@ -40,6 +40,14 @@ OPT(std::string, Backend, "gridtools", "backend", "",
     "\n - c++-naive   = naive C++ code\n"
     "\n - c++-opt     = optimized C++ code"
     "\n - cuda        = optimized cuda", "<backend>", true, false)
+OPT(std::string, arch, "", "arch", "",
+    "Set the computing architecture. Possible values for cuda backend are:"
+    "\n - K80\n"
+    "\n - P100\n", "<arch>", true, false)
+OPT(std::string, domain_size, "", "domain-size", "",
+    "domain size for compiler optimization", "", true, false)
+OPT(std::string, block_size, "", "block-size", "",
+        "block size for compiler optimization", "", true, false)
```

Comment:
non-specific comment. Specify what the meaning of the block-size is

Reply:
done